akka-cluster/src/main/resources/reference.conf

```diff
@@ -65,6 +65,11 @@ akka {
     # move 'WeaklyUp' members to 'Up' status once convergence has been reached.
     allow-weakly-up-members = on
 
+    # Teams are used to make islands of the cluster that are colocated. It can be used
+    # to make the cluster "dc-aware", run the cluster in multiple availability zones or regions.
```

Comment:
spell it out data center aware, and run the cluster across multiple...

or do one say "run a cluster spanning multiple"?